dbms/src/TestUtils/mockExecutor.h

```diff
@@ -96,6 +97,13 @@ class DAGRequestBuilder
     DAGRequestBuilder & aggregation(ASTPtr agg_func, ASTPtr group_by_expr);
     DAGRequestBuilder & aggregation(MockAsts agg_funcs, MockAsts group_by_exprs);
 
+    // window
+    DAGRequestBuilder & window(ASTPtr window_func, MockOrderByItem order_by, MockOrderByItem partition_by, MockWindowFrame frame);
+    DAGRequestBuilder & window(MockAsts window_funcs, MockOrderByItems order_by_list, MockOrderByItems partition_by_list, MockWindowFrame frame);
```

Comment:
can we use a new type `MocPartitionByItems` for `partition_by_list`?

Reply:
Good idea!